src/esp/physics/PhysicsManager.cpp

```diff
@@ -109,9 +109,9 @@ int PhysicsManager::addObject(const std::string& configFileHandle,
           ->getInitializationAttributes()
           ->getIsVisible()) {
     resourceManager_.addObjectToDrawables(
-        configFileHandle, existingObjects_.at(nextObjectID_)->visualNode_,
-        drawables, existingObjects_.at(nextObjectID_)->visualNodes_,
-        lightSetup);
+        existingObjects_.at(nextObjectID_)->getInitializationAttributes(),
```

Comment:
Nit: create a temp var for the result of `existingObjects_.at(nextObjectID_)` instead of doing 3 lookups and copy-pasting code.